demo/deploy-mysql.sh

```diff
@@ -49,7 +49,7 @@ spec:
       nodeSelector:
         kubernetes.io/os: linux
       containers:
-      - image: mysql:5.6
+      - image: mysql/mysql-server:8.0.28
```

Comment:
@allenlsy Are there any concerns with bumping the MySQL version as long as the CI passes?